packages/components/tests/helpclasses/base-test-help.js

```diff
@@ -17,414 +18,416 @@ import AuthModule from '@/vue/vuex-modules/data/auth-module.js'
-        getFeatureOptions: () => { return BaseTestHelp.defaultFeatureOptions },
-        getResourceOptions: () => { return BaseTestHelp.defaultResourceOptions },
-        getUiOptions: () => { return BaseTestHelp.defaultUIOptions },
-        verboseMode: () => { return false }
-      }
-      return Object.assign(defaultProps, props)
+    return Object.assign(defaultProps, props)
+  }
+
+  static settingsAPI (props) {
+    const defaultProps = {
+      lookupResourceOptions: BaseTestHelp.defaultResourceOptions,
+      getFeatureOptions: () => { return BaseTestHelp.defaultFeatureOptions },
+      getResourceOptions: () => { return BaseTestHelp.defaultResourceOptions },
+      getUiOptions: () => { return BaseTestHelp.defaultUIOptions },
+      verboseMode: () => { return false }
     }
-
-    static appAPI (props) {
-      let defaultProps = {
-        platform: {
-          viewport: {
-            width: 0,
-            height: 0
-          }
-        },
-        state: {
-          lemmaTranslationLang: 'lat',
-          selectedLookupLangCode: 'lat'
-        },
-        wordUsageExamples: null,
-
-
-        hasMorphData: () => false,
-        getHomonymLexemes: () => null,
-        getDefaultLangCode: () => 'lat',
-        getLanguageName: UIController.getLanguageName,
-
-        updateLanguage: () => true,
-        enableWordUsageExamples: () => true,
-        setSelectedLookupLang: () => true,
-
-        newLexicalRequest: () => true
-      }
-      return Object.assign(defaultProps, props)
+    return Object.assign(defaultProps, props)
+  }
+
+  static appAPI (props) {
+    const defaultProps = {
+      platform: {
+        viewport: {
+          width: 0,
+          height: 0
+        }
+      },
+      state: {
+        lemmaTranslationLang: 'lat',
+        selectedLookupLangCode: 'lat'
+      },
+      wordUsageExamples: null,
+
+      hasMorphData: () => false,
+      getHomonymLexemes: () => null,
+      getDefaultLangCode: () => 'lat',
+      getLanguageName: UIController.getLanguageName,
+
+      updateLanguage: () => true,
+      enableWordUsageExamples: () => true,
+      setSelectedLookupLang: () => true,
+
+      newLexicalRequest: () => true
     }
-
-
-    static authModule (store, api) {
-      return new AuthModule(store, api, { auth: null })
+    return Object.assign(defaultProps, props)
+  }
+
+  static authModule (store, api) {
+    return new AuthModule(store, api, { auth: null })
+  }
+
+  static l10nModule (store, api) {
+    return new L10nModule(store, api, {
+      defaultLocale: Locales.en_US,
+      messageBundles: Locales.bundleArr([
+        [enUS, Locales.en_US],
+        [enUSData, Locales.en_US],
+        [enUSInfl, Locales.en_US],
+        [enGB, Locales.en_GB]
+      ])
+    })
+  }
+
+  static lexisModule (store, api) {
+    return new LexisModule(store, api, {
+      name: 'cedict',
+      targetURL: 'https://lexis-dev.alpheios.net/index-dev.html',
```

Comment:
does this create a test-time dependency on the service running there?

Reply:
This is an important consideration and I agree with your concerns. It is not used in tests now (i.e. we're not testing a remote connectivity of Lexis CS at the moment) so I removed those config props. I'm not sure what would be the best way, but maybe we should use fixtures for things like this in the future.

Reply:
probably yes